uptime-kuma/templates/pvc.yaml

```diff
@@ -11,6 +11,8 @@ metadata:
     {{- include "uptime-kuma.labels" . | nindent 4 }}
     {{- with .Values.persistence.labels }}
     {{- toYaml . | nindent 4 }}
+    {{- with .Values.commonLabels }}
+    {{- toYaml . | nindent 4 }}
```

Comment:
same, the include should already cover the `commonLabels` :thinking: 

and seems like there are more instances of this, but I will stop here for now

Reply:
You are right, i forgot I added it to the _helpers.tpl